### PR TITLE
Package vocal.0.1

### PR DESCRIPTION
**Pull request by mariojppereira**
### `vocal.0.1`
VOCaL -- The Verified OCaml Library
VOCaL -- The Verified OCaml Library

---
* Homepage: https://vocal.lri.fr/
* Source repo: git://github.com/vocal-project/vocal
* Bug tracker: https://github.com/vocal-project/vocal/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
there is no way this package is getting merged until the authors names gets fixed. Over my dead body. I'll fix it myself later today.

File: packages/vocal/vocal.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+license: "MIT"
+homepage: "https://vocal.lri.fr/"
+dev-repo: "git://github.com/vocal-project/vocal"
+bug-reports: "https://github.com/vocal-project/vocal/issues"
+doc: "https://github.com/vocal-project/vocal/blob/master/README.md"
+build: [
+  [ "dune" "build" "-p" name ]
+]
+depends: [
+  "dune" {build}
```
[the remark this answers is not in the document]
Comment (kit-ty-kate) [reply packages/vocal/vocal.0.1/opam]: it's always good to have it indeed. Fixed

**Comment by kit-ty-kate** [comment]
cc @rgrinberg

**Comment by kit-ty-kate** [comment]
Just waiting to clear something up. Please don't merge.

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by rgrinberg** [comment]
Dune needs to hard code the presence of seq for 4.07+. Sigh...

Thanks for notifying me, I will make a fix. In the meantime, I recommend just using the seq package.

**Pull request merged**